reactapp/src/redux/actions.js

```diff
@@ -0,0 +1,27 @@
+export function ADD_AUTH(id, email, name) {
```

Comment:
Let's use uppercase naming for constants. functions/methods should be camelCased